src/Microsoft.Health.Fhir.Liquid.Converter.Tool/ConverterLogicHandler.cs

```diff
@@ -16,12 +16,23 @@
 
 namespace Microsoft.Health.Fhir.Liquid.Converter.Tool
 {
-    internal static class ConverterLogicHandler
+    public static class ConverterLogicHandler
     {
         private const string MetadataFileName = "metadata.json";
         private static readonly List<string> CcdaExtensions = new List<string> { ".ccda", ".xml" };
         private static readonly ProcessorSettings DefaultProcessorSettings = new ProcessorSettings();
 
+        public static string ConvertWithoutSaving(string templateDirectory, string rootTemplate, string inputDataContent, bool isVerboseEnabled, bool isTraceInfo)
```

Comment:
what's the thinking behind the spelled out args here vs using a `ConverterOptions` object?